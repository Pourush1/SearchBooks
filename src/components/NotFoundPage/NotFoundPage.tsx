import React from 'react';

const NotFoundPage: React.FC<{}> = () => {
  return (
    <div className="text-center">
      <h1 className="text-warning">Page Not Exists</h1>
      <p className="text-warning">404</p>
    </div>
  );
};

export default NotFoundPage;
