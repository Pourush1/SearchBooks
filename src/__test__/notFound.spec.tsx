import React from 'react';
import { render, RenderResult } from '@testing-library/react';
import NotFoundPage from '../components/NotFoundPage/NotFoundPage';

let documentBody: RenderResult;

describe('<NotFound />', () => {
  beforeEach(() => {
    documentBody = render(<NotFoundPage />);
  });

  it('matches snapshot', () => {
    const { baseElement } = documentBody;
    expect(baseElement).toMatchSnapshot();
  });
});
