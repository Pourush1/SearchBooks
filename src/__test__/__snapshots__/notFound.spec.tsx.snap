// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> matches snapshot 1`] = `
<body>
  <div>
    <div
      class="text-center"
    >
      <h1
        class="text-warning"
      >
        Page Not Exists
      </h1>
      <p
        class="text-warning"
      >
        404
      </p>
    </div>
  </div>
</body>
`;
